### DISCUSSION TITLE
CMAKE_CXX_FLAGS=" -Ofast -march=znver1 "

### DISCUSSION THREAD

**Discussion by noskoski** [madMAx43v3r/chia-plotter#516] [Ideas]
Supposedly I got 10% less time using AMD Ryzen with CMAKE_CXX_FLAGS="-Ofast -fmax-errors=1 -march=znver1 ", from ~3000 seconds to ~2800, anyone else also had similar results ?

**Comment by vladaepro** [comment]
yes, i got some speedup with -Ofast -march=native -mtune=native
